Makefile

```diff
@@ -71,10 +71,11 @@ oapigen: install-oapi-codegen
 .PHONY: generate
 generate:
 	@wget -q --show-progress --progress=dot https://openapi-v2.exoscale.com/source.yaml -O- > v3/generator/source.yaml
+	@echo
 	@cd v3/generator/
 	@go generate
 	@go mod tidy
 	@go mod vendor
 	@rm source.yaml
 	@cd ..
-	@ls -l *.go
+	@ls -l *.go 1>&2
```

Comment:
Added it to be able to run `GENERATOR_DEBUG` without noise.
e.g: `GENERATOR_DEBUG=schemas make generate > test/schemas.go`

 I'll add it debug instruction in the README